src/redux/actions/reportData.js

```diff
@@ -45,7 +45,6 @@ export const loadingReportDataError = () => ({
  * @returns {Function}
  */
 export const loadingReportDataStartWithFeedback = () => dispatch => {
```

Comment:
This one should dispatch `loadingReportDataStart` as well. If there's a reason against dispatching the `loadingReportDataStart` action, then this doesn't need to be a thunk, I'd say?

Reply:
Good one. I removed this to stop showing the spinner overlay, because the table has its own loader. But I shouldn't have changed this thunk. Instead I should have updated the `loadReportData` thunk in `./redux/actions/reportingRateSummary.js`. Updated this now.